File: C++ Programs/Searching Algorithms/Linear Search/CPP_Linear_Search_Basic-main/Linear Search.cpp
   // Linear Search with 1D Array in C++
#include<iostream>
using namespace std;
int main()
{
	int arr[5];
	for(int i=0; i<=4; i++){
		cout<<"Enter Element: ";
		cin>>arr[i];
	}
	int search;
	cout<<"Enter value to search: ";
	cin>>search;
	for(int i=0; i<=4; i++){
		if(search == arr[i]){
			cout<<"Element Found!"<<endl;
			cout<<"Index: "<<i;
			return 0;
		}
	}
	cout<<"Element Not Found!"<<endl;
}

File: Java Programs/Searching Algorithms/Linear Search/Java_Linear_Search_Basic-main/linearSearch.java
import java.util.Scanner;
public class linearSearch {
    public static void main(String[] args) {
        boolean found=false;
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter Value of N: ");
        int n=sc.nextInt();
        int arr[]= new int[n];
        for(int i=0; i<n; i++){
            System.out.print("Enter element: ");
            arr[i]=sc.nextInt();
        }
        System.out.print("Enter element to Search: ");
        int search=sc.nextInt();
        for(int i=0; i<n; i++){
            if(search == arr[i]){
                System.out.println("Element Found!");
                System.out.println("Index: "+i);
                System.out.println("Element: "+arr[i]);
                found=true;
                System.exit(0);
            }
        }
        if(!found){
            System.out.println("Not found!");
        }
    }
}
